tests/cloner_test.go

```diff
@@ -94,10 +95,16 @@ func doCloneTest(f *framework.Framework, targetNs *v1.Namespace) {
 	targetPod, err := utils.FindPodByPrefix(f.K8sClient, targetNs.Name, common.CLONER_TARGET_PODNAME, common.CDI_LABEL_SELECTOR)
 	Expect(err).ToNot(HaveOccurred())
 
-	By("Source and Target pods have to be on same node")
-	fmt.Fprintf(GinkgoWriter, "INFO: Source POD host %s\n", sourcePod.Spec.NodeName)
-	fmt.Fprintf(GinkgoWriter, "INFO: Target POD host %s\n", targetPod.Spec.NodeName)
-	Expect(sourcePod.Spec.NodeName).To(Equal(targetPod.Spec.NodeName))
+	By("Verifying that the source and target pods are scheduled on the same node")
+	Eventually(func() bool {
+		srcNode, err := utils.PodNode(f.K8sClient, sourcePod.Name, sourcePod.Namespace)
```

Comment:
do you mean `PodGetNode`?

Reply:
Ugh... will fix this now in a follow-up.  Trying to work a bit too quickly but I do wonder how it actually compiled for me.

Reply:
And the CI was happy as well...